src/melee2.cc

```diff
@@ -6975,135 +6966,128 @@ static void process_monster(int m_idx, bool_ is_frien)
 					disturb(0);
 			}
 
-			/* Check for monster trap */
-			if (c_ptr->feat == FEAT_MON_TRAP)
-			{
-				if (mon_hit_trap(m_idx)) return;
-			}
-			else
-			{
-				/* Copy list of objects; we need a copy because we're mutating the list. */
```

Comment:
Could would please restore indentation here? (By leaving the surrounding "{".) It's really hard to see what changed.